javascript_nodejs/10.prompt-validations/dialogs/onboard/index.js

```diff
@@ -0,0 +1,114 @@
+const { ComponentDialog, WaterfallDialog } = require('botbuilder-dialogs');
+const moment = require('moment');
+
+// Import a few specialized prompt classes.
+const NamePrompt = require('../../prompts/namePrompt');
+const AgePrompt = require('../../prompts/agePrompt');
+const DOBPrompt = require('../../prompts/dobPrompt');
+const ColorPrompt = require('../../prompts/colorPrompt');
```

Comment:
Not sure where we landed on the placement of these classes... @sgellock, you and I should chat about this tomorrow...